testing/trino-product-tests/src/main/java/io/trino/tests/product/kudu/TestKuduConnectoKerberosSmokeTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.kudu;
+
+import io.trino.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.KUDU;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+
+public class TestKuduConnectoKerberosSmokeTest
```

Comment:
@wendigo do you know the rational behind splitting the product tests into suites?

I'm unsure which suite to put the kudu product tests under and you seem to be a person who built the original suite automation:
https://github.com/trinodb/trino/pull/4895

Reply:
Please put it under Suite7NonGeneric